hardhat_scripts/staging_redeem_queue_migration/redeem_queue_repair.ts

```
@@ -0,0 +1,380 @@
+  open: boolean;
+}
+
+/// Retry transport failures. Tenderly drops the connection under long call bursts, which surfaces as
+/// ECONNRESET / SERVER_ERROR rather than a revert, so a plain retry clears it.
+export async function retry<T>(fn: () => Promise<T>, attempts = 5, delayMs = 400): Promise<T> {
+  let last: any;
+  for (let i = 0; i < attempts; ++i) {
+    try {
+      return await fn();
+    } catch (e: any) {
+      const transport = ["SERVER_ERROR", "TIMEOUT", "NETWORK_ERROR"].includes(e?.code)
```

ESLint

[error] Replace `·["SERVER_ERROR",·"TIMEOUT",·"NETWORK_ERROR"].includes(e?.code)` with `⏎········["SERVER_ERROR",·"TIMEOUT",·"NETWORK_ERROR"].includes(e?.code)·||`

(prettier/prettier)